packages/python-modules/cryptography36/cryptography-py27-warning.patch

```diff
@@ -0,0 +1,14 @@
+Delete the warning that breaks tests of dependent projects.
+
+--- a/src/cryptography/__init__.py
++++ b/src/cryptography/__init__.py
+@@ -33,9 +32,0 @@ __all__ = [
+-
+-if sys.version_info[0] == 2:
```

Comment:
Why do we need this patch? The code above seem to be using python3 and here we are removing a check testing whether the user tries to use python2.

Reply:
cryptography36 was copied directly from an older nixpkgs. I guess it would be fine if i removed the patch. It's simply there cause it was in the cryptography in nixpkgs.